kartothek/io/dask/delayed.py

```diff
@@ -230,6 +283,91 @@ def merge_datasets_as_delayed(
     return list(mps)
 
 
+@default_docs
+def merge_many_datasets_as_delayed(
+    dataset_uuids: List[str],
+    store,
+    merge_tasks,
+    match_how="exact",
+    dispatch_by=None,
+    label_merger=None,
+    metadata_merger=None,
+    predicates=None,
+    columns: Optional[List[Dict[str, List[str]]]] = None,
+):
+    """
+    A dask.delayed graph to perform the merge of two full kartothek datasets.
+
+    Parameters
+    ----------
+    dataset_uuids : List[str]
+    match_how : Union[str, Callable]
+        Define the partition label matching scheme.
+        Available implementations are:
+
+        * first : The partitions of the first dataset are considered to be the base
+                  partitions and **all** partitions of the remaining datasets are
+                  joined to the partitions of the first dataset. This should only be
+                  used if all but the first dataset contain very few partitions.
```

Comment:
What does "few" mean? What happens if this is not the case? Please give the user more guidance and try to provide a more failure-proof API.